proto/topic_eth_signature_message.proto

```diff
@@ -10,4 +10,5 @@ message TopicEthSignatureMessage {
   string amount = 3;
   string fee = 4;
   string signature = 5;
+  string asset = 6;
```

Comment:
I think that here we should specify that it is `erc20Address`, because that is the data that will be signed in the signature